src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -262,7 +263,91 @@ protected set
                 {
                     foreach (View child in Children)
                     {
-                        child.ControlState = value;
+                        ControlState Allowed = child.PropagatableControlStates;
```

Comment:
Just my curiosity :
Is `Allowed` a grammar of C# or just a variable here?
If it's a variable of ControlState, how about changing its name to `allowed`?

Reply:
sorry for the confusion. yes it should be renamed allowed. let me update quickly.